### ISSUE TITLE
Media Queries Problem

### ISSUE THREAD

**Issue by cieraclaire** [cieraclaire/restaurant-website#4]
I honestly can't figure out why my media queries screw up every time I size down to mobile (which is looking good) then size it back up to tablet and desktop. It all mixes up. (I may have figured it out but we shall see when you open it up).

ALSO, since I completely re-wrote my code 4 times, I am stuck with this again: Functionality - index.html the website isn't functioning as expected: background colour of nav links doesn't change when hovered. I completely forget what you told me to do to fix that. That is the only thing that is keeping me from re-submitting.

@thomasjbradley

**Comment by thomasjbradley** [comment]
_Media queries:_

The media queries seem to be doing things—is there a specific problem you're running into with them?
The CSS for each media query is applying.

---

_Functionality:_

Have you synced and committed that? When I run your website through Markbot it seems to be okay.

**Comment by cieraclaire** [comment]
I don't even know what happened, but yeah it all seems to be working fine now! Thank goodness [😊]  And you're just using the url on canvas of our website to grade correct? I don't need to link to anything? I already submitted my request for grade on canvas as well.

**Comment by thomasjbradley** [comment]
Yeah, the URL that Markbut submits on Canvas is the one I'll be looking at.
